pkg/messages/treaty.go

```diff
@@ -0,0 +1,95 @@
+package messages
+
+import (
+	"github.com/google/uuid"
+)
+
+type ConditionType int // type of condition
+type RequestType int
+type Op int
+
+const (
+	HP ConditionType = iota + 1
+	Floor
+	AvailableFood
+)
+
+const (
+	LeaveAmountFood RequestType = iota + 1
+	LeavePercentFood
+	Inform
+)
+
+const (
+	GT Op = iota + 1
+	GE
+	EQ
+	LE
+	LT
+)
+
+type Treaty struct {
+	condition      ConditionType
```

Comment:
```suggestion
        proposerID   uuid.UUID
        responderID uuid.UUID
	condition      ConditionType
```

Would need these to keep track of who the treaty is between for opinion formation

Reply:
sorry for the formatting eeeew

Reply:
Done!